tensorflow/tensorflow.bzl

```diff
@@ -209,6 +210,7 @@ def tf_copts(android_optimization_level_override="-O2", is_external=False):
       + if_tensorrt(["-DGOOGLE_TENSORRT=1"])
       + if_mkl(["-DINTEL_MKL=1", "-DEIGEN_USE_VML"])
       + if_mkl_lnx_x64(["-fopenmp"])
+      + if_mkl_darwin(["-Xpreprocessor", "-fopenmp"])
```

Comment:
`-fopenmp` used to be listed under `if_mkl` but we had to remove it from there because it failed on macos with the following error:

```
clang: error: unsupported option '-fopenmp'
```

Reply:
If you used if_mkl_darwin(["-Xpreprocessor", "-fopenmp"]), not just "-fopenmp", the errors would disappear. In clang you need to add -Xpreprocessor to use openmp at least Mac OS version 10.13

Reply:
I just patched your PR, and after standard configuration tried the following:
```
bazel test --config=mkl tensorflow/python/kernel_tests:summary_ops_test
```
The result is:
```
external/eigen_archive/unsupported/Eigen/CXX11/../../../Eigen/Core:275:10: fatal error: 'omp.h' file not found
#include <omp.h>
         ^~~~~~~
```

So while it clears the "-fopenmp" not found issue, even with "-Xpreprocessor" omp header is not found.

Reply:
I tried your test command got the results passed as below in our local environment.
I guessed  you used xcode version clang that would meet this errors. Our PR is to support default macos toolchain using openmp. seems you need to switch the command line toolchain . Here are some tips to help you.

1.	Open the “Terminal.app”, type “xcode-select --install”, and click “install” in the popup window.
2.	Type “sudo xcode-select -s /Library/Developer/CommandLineTools” to switch the build chain for cmd.


Target //tensorflow/python/kernel_tests:summary_ops_test up-to-date:
  bazel-bin/tensorflow/python/kernel_tests/summary_ops_test
INFO: Elapsed time: 2062.925s, Critical Path: 193.76s
INFO: Build completed successfully, 1565 total actions
//tensorflow/python/kernel_tests:summary_ops_test                        PASSED in 5.6s

Executed 1 out of 1 test: 1 test passes.

Reply:
Most people are using the default Xcode toolchain for building on macos.
We need to have this work with the default toolchain, otherwise tensorflow team has to field dozens of github issues with this workaround.
Therefore, I will reject this PR, and for the lines in question, I would like to add extra preprocessor directives to avoid using `omp.h` on macos.

Reply:
ok, we will try to resolve it. thanks.